runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1378,16 +1391,17 @@ func s:HandleCursor(msg)
       endif
       exe lnum
       normal! zv
-      exe 'sign unplace ' . s:pc_id
-      exe 'sign place ' . s:pc_id . ' line=' . lnum . ' name=debugPC file=' . fname
```

Comment:
prio was added in 8.2.1556 which isnt ported, but should be fine

Reply:
Since it was already used in the old line 1485 below, I would think so, too ;)